### DISCUSSION TITLE
[A-L] (2023/24) Foglio 10 - Esercizio 2

### DISCUSSION THREAD

**Discussion by Elia-Belli** [sapienzastudentsnetwork/algebra#327] [Esercizi - Viaggi/Piazza] [answered]
![Screenshot 2023-11-21 105404](https://github.com/sapienzastudentsnetwork/algebra2324/assets/107944476/6fba3bd4-a720-4618-a6f8-85ad688d99d2)

**Comment by Elia-Belli** [comment]
Si nota subito che $\mathbb{R}^2$ non è uno spazio vettoriale col prodotto per scalare così definito, perchè il prodotto per l'elemento neutro $1$ in $\mathbb{R}$ deve restituire il vettore invariato, mentre:
$$1(x,y)=(1x,-1y)=(x,-y)\neq (x,y)$$

**Comment by CiottoloMaggico** [comment]
![image](https://github.com/sapienzastudentsnetwork/algebra2324/assets/79638284/72ba3385-ad18-4da9-ae19-49666b921709)

**Comment by Elia-Belli** [comment, marked as answer]
# Soluzione Ufficiale
![Screenshot 2023-11-29 112508](https://github.com/sapienzastudentsnetwork/algebra2324/assets/107944476/98fc9fec-8baf-4f8f-a611-2a85719eafb8)
